src/relay/config.ts

```diff
@@ -1,9 +1,7 @@
 import * as Relay from "react-relay"
 
-export const metaphysicsURL = "https://metaphysics-staging.artsy.net"
-
 export function artsyNetworkLayer(user?: any) {
-  return new Relay.DefaultNetworkLayer(metaphysicsURL, {
+  return new Relay.DefaultNetworkLayer(process.env.METAPHYSICS_ENDPOINT, {
```

Comment:
Unfortunately, we can't use `process.env` in this case because this module is also used on the client-side.
An alternative would be to use sharify to set the metaphysicsURL